instances/widgets.treasury-factory.near/widget/pages/asset-exchange/CreateRequest.jsx

```diff
@@ -142,6 +139,27 @@ return (
       showInProgress={isTxnCreated}
       cancelTxn={() => setTxnCreated(false)}
     />
+
+    {/* Treasury Wallet Dropdown - Only show if showNearIntents is enabled */}
+    {showNearIntents && (
+      <div className="mb-3">
```

Comment:
do we want this feature as a feature flag? because then it would make sense to have this in payment requests as well, which I m working on

Reply:
I think we can release payment requests without the feature flag ( it requires that the deposits are with the same token / network as the payment request ). This asset exchange feature, we should still keep behind the feature flag, since we need more manual testing with the 1click API, and there are still known issues that we need to implement ( like the JWT auth, and preventing approving proposals with expired quotes ).